### ISSUE TITLE
Server returns wrong download url when behind Nginx reverse proxy

### ISSUE THREAD

**Issue by chakflying** [timvisee/send#29]
so in Nginx config, I gave the Send server a name like this:

```
upstream send_backend {
    server 127.0.0.1:1443;
}
...
    proxy_pass http://send_backend
```

The problem is now when the user finishes uploading a file, the url shown is `https://send_backend/download...`, which obviously doesn't work. 

I took a quick look at the code and seems like `req.get('host')` is what's causing the problem. Is it possible for it to use the `BASE_URL` in config / environment variable instead? Or have I configured Nginx wrong somehow?

**Comment by timvisee** [comment]
It looks like you're right.

I believe setting an explicit `Host` header should work, see: https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/#passing-request-headers

I've merged https://github.com/timvisee/send/pull/24 minutes ago which allows you to set whether to use a dynamic host (like it is currently doing), but it doesn't seem to handle this specific case yet though. I'll change that now.

**Comment by timvisee** [comment]
Send now properly uses the configured base URL for upload links as of 4c45d6217d0fc1b3031410d3f4983b36f9eeb687, that should solve this issue.

I've released this as part of `v3.4.10`. It should be available as soon as this release pipeline succeeds: https://gitlab.com/timvisee/send/-/pipelines/299036714
